src/authentication/helpers/get-profile-info.ts

```diff
@@ -92,7 +92,11 @@ export function getProfileStamboekNumber(user: Avo.User.User | undefined): strin
 export function isProfileComplete(user: Avo.User.User): boolean {
 	const profile = get(user, 'profile');
 
-	// TODO implement check based on user role
+	if (user.role_id !== 2) {
```

Comment:
Ik zou eerder de comment hierboven plaatsen.

`// Students do not have to fill in their profile`

dan eronder omdat het logischer leest. 🙈

Reply:
er zijn nog meer gebruikersgroepen dan enkel lesgevers en leerlingen
ik zal deze comment erboven zetten:

```
// Only teachers have to fill in their profile for now
```